django/db/models/functions/text.py

```diff
@@ -185,6 +185,21 @@ def __init__(self, expression, text, replacement=Value(''), **extra):
         super().__init__(expression, text, replacement, **extra)
 
 
+class Reverse(Transform):
+    function = 'REVERSE'
+    lookup_name = 'reverse'
+
+    def as_oracle(self, compiler, connection, **extra_context):
+        return super().as_sql(
```

Comment:
Should we add a brief comment here explaining why we are using `XMLQUERY`/the problems with the standard `reverse` function?

Reply:
Sure, I can add a brief comment.

Reply:
Done.